rero_ils/modules/documents/jsonschemas/documents/document_entity_local-v0.0.1.json

```diff
@@ -45,11 +71,114 @@
     },
     "identifiedBy": {
       "$ref": "https://bib.rero.ch/schemas/documents/document_identifier-v0.0.1.json#/identifier_contribution"
+    },
+    "source": {
+      "title": "Source",
+      "description": "Source of the subject.",
+      "type": "string",
+      "minLength": 3,
+      "form": {
+        "templateOptions": {
+          "itemCssClass": "col-lg-6"
+        }
+      }
+    },
+    "subdivisions": {
+      "title": "Subdivisions",
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "items": {
+        "type": "object",
+        "title": "Entities",
+        "description": "Topic (including genre/form), place, temporal, person, family or corporate body (including conferences). Always create a link to IdRef or GND, if possible.",
```

Comment:
"Topic (including genre/form), place, temporal, person, family or corporate body (including conferences). Always create a link to IdRef or GND, if possible."

Reply:
Seems a copy from the original description.